velox/common/memory/MemoryPool.cpp

```diff
@@ -408,11 +408,13 @@ MemoryPoolImpl::~MemoryPoolImpl() {
 
   if (isLeaf()) {
     if (usedReservationBytes_ > 0) {
+      LOG(ERROR) << "Bad memory usage track state : " << toString();
       RECORD_METRIC_VALUE(
           kMetricMemoryPoolUsageLeakBytes, usedReservationBytes_);
     }
 
     if (minReservationBytes_ > 0) {
+      LOG(ERROR) << "Bad memory usage track state : " << toString();
```

Comment:
```
Memory reservation leak: 
```

Reply:
Used this instead to make it consistent with the used memory leak message:
> Memory leak (Reserved memory):